robottelo/hosts.py

```diff
@@ -598,7 +598,7 @@ def reset_rhsm(self):
         """Global Registration points the host's sub-man to talk to the Sattelite's Candlepin
         but saves the original rhsm.conf. Reset the rhsm.conf so that it points back to the CDN.
         """
-        self.execute(r'\cp -f /etc/rhsm/rhsm.conf{.bak,}')
+        self.execute(r'\cp -f /etc/rhsm/rhsm.conf{,.bak}')
```

Comment:
This was originally added in https://github.com/SatelliteQE/robottelo/pull/15931 by @vsedmik .
So I tested this here, so I know it exists on the systems that are registered to Satellite, and this changes it back to be able to get registered to CDN again. But it could be not present on new system which are not registered

Reply:
@Gauravtalreja1 I see, I hit the error I mentioned while stepping through an upgrade scenario and misinterpreted the context. Thanks for the clarification. I'll close this PR now.